### PR TITLE
Footer + meta

### LINKED ISSUE #560: Review styles code + footer text
- [x] Change color in code tags
- [x] Add subtitle in footer content
- [x] Meta tags

### PR DESCRIPTION
**Pull request by israelperezglez**
@raulraja @pakoito could you review this PR please?

Fix #560

### CODE REVIEW

File: arrow-docs/docs/_includes/_head.html [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+   <meta name=“keywords” content=“{{site.data.commons.keywords}}“>
 
-    <meta property="og:image" content="http://arrow-kt.io/img/poster.png" />
-    <meta property="og:title" content="{{site.data.commons.name}}" />
-    <meta property="og:site_name" content="{{site.data.commons.name}}" />
-    <meta property="og:url" content="http://arrow-kt.io/" />
-    <meta property="og:type" content="website" />
-    <meta property="og:description" content="{{site.data.commons.description}}" />
-    <meta property="og:keywords" content="{{site.data.commons.keywords}}" />
+  <meta property=“og:image” content=“http://arrow-kt.io/img/poster.png” />
```
Comment (pakoito) [inline arrow-docs/docs/_includes/_head.html]: These quotes aren't the supported ones, I believe.

**Comment by israelperezglez** [comment]
@pakoito Thanks! could you review please?

**Comment by jorgegalindocruces** [comment]
@pakoito good morning in the morning buddy!
Simple question, what the project needs to deploy the website?

This change affects the Twitter card visualization (if anyone links the website on Twitter)

**Comment by jorgegalindocruces** [comment]
AH! Just saw it! Nevermind ;-)

**Pull request merged**